src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+    protected boolean isDone;
+    protected String description;
+    protected static int noOfTasks = 0;
+    protected String taskIcon;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+        this.taskIcon = taskIcon;
+        noOfTasks++;
+    }
+
+    public String getDescription() {
```

Comment:
I like the use of camelCase for methods and PascalCase for class names. Great job!